src/index.html

```diff
@@ -11,7 +11,182 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form method="get"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <div class="first box">
+        <fieldset class="first">
+          <legend>
+            Personal information
+          </legend>
+          <div>
+            <label>
+              Surname:
+              <input type="name"
+              autocomplete="off"
+              name="usersurname"
+              >
+            </label>
+          </div>
+          <div class="fieldelement">
+            <label
+            >
+              Name:
+              <input type="name"
+              autocomplete="off"
+              name="username"
+              >
+            </label>
+          </div>
+          <div class="fieldelement">
+            <label>
+              How old are You?
+              <input type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+              >
+            </label>
+          </div>
+          <div class="fieldelement">
+            <label>
+              Full date of birth:
+              <input type="date"
+              name="birth"
+              >
+            </label>
+          </div>
+          <div class="fieldelement">
+            <label>
+            I accept the term of the agreement
+              <input type="checkbox"
+              name="agreement"
+              >
```

Comment:
for two or less attributes normal to keep in one line
```suggestion
              <input type="checkbox" name="agreement">
```